ui/src/main/resources/Meeting/Code/MeetingSheet.xml

```diff
@@ -218,6 +218,12 @@
           ## poll
           #if($doc.isCurrentUserCreator() &amp;&amp; $xwiki.exists('XPoll.WebHome')
               &amp;&amp; $doc.getValue('status') == 'value1' &amp;&amp; $xcontext.action == 'view')
+            #if($doc.getValue('hasPoll') == 1)
```

Comment:
You should use an `if-else` statement instead of  checking for both `==1` and `!=1`.